src/main/java/nl/knaw/dans/migration/cli/LoadFromFedoraCommand.java

```diff
@@ -63,7 +64,14 @@ public LoadFromFedoraCommand(
     @Override
     public void configure(Subparser subparser) {
         super.configure(subparser);
-        subparser.addArgument("csv")
+        subparser.addArgument("-f", "--withFiles")
+            .dest(WITH_FILES)
+            .type(Boolean.class)
+            .setConst(true)
+            .setDefault(false)
+            .nargs("?")
+            .help("The table expected_files is not filled without this option");
```

Comment:
The javadoc of `nargs`seems to suggest a configuration for options without a required value, but I'm afraid the positional arguments make it impossible to specify `--withFiles` without a value `true` or `false`.